builders/toolkit/ethereum-api/precompiles/cross-chain-transfers.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Native Cross-Chain Token Transfers
+description: Learn how to use the XCM interface precompile to transfer tokens from any Tanssi EVM appchain, leveraging their inherent native cross-chain capabilities.
+---
+
+# Native Cross-Chain Token Transfers
+
+## Introduction {: #introduction }
+
+As presented in the [Native Cross-Chain Communication](/learn/framework/xcm/){target=\_blank} article from the Learn section, Tanssi appchains benefit from an inherent capability to communicate and interoperate with any other appchain in the ecosystem. This native cross-chain communication allows safe and fast token transfers leveraging the Cross-Consensus Message format (XCM for short), which facilitates communication between different consensus systems.
+
+The communication protocol enabling token transfers is built on [Substrate](/learn/framework/overview/#substrate-framework){target=\_blank} and runs on a lower level than the EVM, making it harder for EVM developers to access.
+
+Nevertheless, EVM Appchains have an XCM precompile that fills the gap between execution layers, exposing a smart contract interface that abstracts away the underlying complexities, making the execution of cross-chain token transfers as easy as any other smart contract call.
+
+This guide will show you how to interact with the [XCM interface](https://github.com/moondance-labs/tanssi/blob/master/test/contracts/solidity/XcmInterface.sol){target=\_blank} precompile to execute cross-chain token transfers through the Ethereum API.
+
+The XCM precompile is located at the following address:
+
+```text
+{{networks.dancebox.precompiles.xcmInterface }}
```

Comment:
The variable was updated in the `variables.yml` file, but also needs to be updated here where you use it